The name of the dynamics module is dynamics_pinocchio
The name of the dynamics class is DynamicPinocchio

--- src/dynamic_graph/sot/pattern_generator/meta_pg.py	2016-10-19 16:38:57.000000000 +0200
+++ src/dynamic_graph/sot/pattern_generator/meta_pg.py	2017-02-28 15:39:47.764456278 +0100
@@ -1,7 +1,7 @@
 from dynamic_graph.sot.core import *
 from dynamic_graph import plug
 from dynamic_graph.sot.pattern_generator import PatternGenerator,Selector
-from dynamic_graph.sot.dynamics import *
+from dynamic_graph.sot.dynamics_pinocchio import *
 
 class MetaPG:
     def __init__(self,dyn):
@@ -44,7 +44,7 @@
         self.pg.initState()
 
         # --- PG INIT FRAMES ---
-        self.geom = Dynamic("geom")
+        self.geom = DynamicPinocchio("geom")
         self.geom.setFiles(modelDir, modelName,specificitiesPath,jointRankPath)
         self.geom.parse()
         self.geom.createOpPoint('rf','right-ankle')


--- src/dynamic_graph/sot/pattern_generator/walking.py	2016-10-19 16:38:57.000000000 +0200
+++ src/dynamic_graph/sot/pattern_generator/walking.py	2017-02-28 15:39:47.764456278 +0100
@@ -1,12 +1,12 @@
 # --- PG ---------------------------------------------------------
 from dynamic_graph import plug
 from dynamic_graph.sot.core.math_small_entities import Derivator_of_Matrix, Inverse_of_matrixHomo, Multiply_of_matrixHomo, Stack_of_vector, PoseRollPitchYawToMatrixHomo, MatrixHomoToPoseRollPitchYaw, Multiply_matrixHomo_vector
-from dynamic_graph.sot.dynamics import Dynamic
+from dynamic_graph.sot.dynamics_pinocchio import DynamicPinocchio
 import dynamic_graph.script_shortcuts
 from dynamic_graph.script_shortcuts import optionalparentheses
 from dynamic_graph.matlab import matlab
 from dynamic_graph.sot.core import *
-from dynamic_graph.sot.dynamics import *
+from dynamic_graph.sot.dynamics_pinocchio import *
 from dynamic_graph.sot.core.meta_task_6d import MetaTask6d,toFlags
 from dynamic_graph.sot.pattern_generator import PatternGenerator,Selector
 from dynamic_graph.sot.core.matrix_util import matrixToTuple
@@ -96,7 +96,7 @@
 
 def addPgTaskToVRMLRobot(robot,solver):
   # --- ROBOT.PG INIT FRAMES ---
-  robot.geom = Dynamic("geom")
+  robot.geom = DynamicPinocchio("geom")
   print("modelDir: ",robot.modelDir)
   print("modelName:",robot.modelName)
   print("specificitiesPath:",robot.specificitiesPath)
@@ -107,7 +107,7 @@
   
 def addPgTaskToUrdfRobot(robot,solver):
     # --- ROBOT.PG INIT FRAMES ---
-    robot.geom = Dynamic("geom")
+    robot.geom = DynamicPinocchio("geom")
     if(hasattr(robot, 'jointMap')):
         for i in robot.jointMap:
             robot.geom.addJointMapping(i, robot.jointMap[i])
